src/wallet/rpcdump.cpp

```diff
@@ -302,8 +302,7 @@ UniValue importprunedfunds(const UniValue& params, bool fHelp)
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
     if (pwalletMain->IsMine(tx)) {
-        CWalletDB walletdb(pwalletMain->strWalletFile, "r+", false);
```

Comment:
The third argument `fFlushOnClose` (`false` here) will be `true` after this PR (default value is `true`).
= before this PR `importprunedfunds` will not flush on close. After this PR it will.
But I guess this is not a problem.